types/src/on_chain_config/aptos_features.rs

```diff
@@ -55,6 +55,7 @@ pub enum FeatureFlag {
     RECONFIGURE_WITH_DKG = 45,
     ZK_ID_SIGNATURE = 46,
     OPEN_ID_SIGNATURE = 47,
+    REFUNDABLE_BYTES = 48,
```

Comment:
Would it be easier if we just gate this change by the gas feature version?

Reply:
I feel I have a higher-than-average possibility of reverting this, when I needed to do it, I don't want to take down other things with me. :p